src/main/java/shortestpath/Transport.java

```diff
@@ -33,9 +33,9 @@ public class Transport {
     @Getter
     private List<Quest> quests = new ArrayList<>();
 
-    /** The ids of items required to use this transport. If the player has **any** matching item, this transport is valid */
+    /** The ids of items required to use this transport. If the player has **any** of the matching list of items, this transport is valid */
     @Getter
-    private List<Integer> itemIdRequirements = new ArrayList<>();
+    private List<List<Integer>> itemIdRequirements = new ArrayList<>();
```

Comment:
This change only effects the existing items tsv. That file already used a list of item ids delimited by `;`. If any of the item ids were found in the player inventory, then that item transport could be used. Effectively, the `;` can be though of as a boolean OR. 

This change adds sub lists delimited by `,` where every item in list must be present in the inventory. The `,` is the boolean AND.

Example: `10,11;12;13,14` means there are 3 possible item id combinations that could be used. Any of these combinations:
- 10 and 11 must be in the inventory
- 12 must be in the inventory
- 13 and 14 must be in the inventory